crypto-dashboard/src/index.css

```
@@ -0,0 +1,1424 @@
+
+button,
+select {
+  text-transform: none;
+}
+
+button,
+[type='button'],
+[type='reset'],
+[type='submit'] {
+  -webkit-appearance: button;
+  appearance: button;
```

Stylelint (17.12.0)

[error] Expected "button" to be "auto" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)